edge/pkg/metamanager/process.go

```diff
@@ -549,12 +545,14 @@ func (m *metaManager) syncPodStatus() {
 			klog.Errorf("unmarshal podstatus[%s] failed, content[%s]: %v", v.Key, v.Value, err)
 			continue
 		}
-		content = append(content, podStatus)
+		contents[namespaceParsed] = append(contents[namespaceParsed], podStatus)
 	}
-
-	msg := model.NewMessage("").BuildRouter(MetaManagerModuleName, GroupResource, namespace+constants.ResourceSep+model.ResourceTypePodStatus, model.UpdateOperation).FillBody(content)
-	sendToCloud(msg)
-	klog.Infof("sync pod status successful, %s", msgDebugInfo(msg))
+	for namespace, content := range contents {
+		msg := model.NewMessage("").BuildRouter(MetaManagerModuleName, GroupResource, namespace+constants.ResourceSep+model.ResourceTypePodStatus, model.UpdateOperation).FillBody(content)
+		sendToCloud(msg)
+		klog.V(3).Infof("sync pod status successfully for namespaces %s, %s", namespace, msgDebugInfo(msg))
+	}
+	klog.Infof("successfully sync pod status in edge-store to cloud")
```

Comment:
Default logs should focus on the error log, this successfully log will be printed out periodly, I think it's useless and can remove it, WDYT?

Reply:
Yes, it sounds all reasonable

> Default logs should focus on the error log, this successfully log will be printed out periodly, I think it's useless and can remove it, WDYT?

Yes, it sounds all reasonable